src/wizard/ActionBox.tsx

```diff
@@ -20,20 +25,20 @@ interface Props {
 export class ActionBox extends Component<Props> {
   render = (): React.ReactElement => {
     return (
-      <Page className={this.props.first ? 'ActionBox' : 'ActionBox RightArrow'}>
-        <div className="ups-number">
+      <Card className={this.props.first ? 'ActionBox' : 'ActionBox RightArrow'}>
+        <div className="ups-number" style={{ textAlign: 'center' }}>
           <span>{this.props.index}</span>
         </div>
-        <PageSection variant={'default'}>
-          <div className={'ActionBox-content'}>
+        <CardHead style={{ marginTop: 15 }}>
+          <div className={'ActionBox-content'} style={{ textAlign: 'center', width: '100%' }}>
```

Comment:
Wouldn't be better to add the styles to the class instead of using the `style` attribute?

Reply:
ok!